app/assets/javascripts/discourse/components/d-button.js.es6

```diff
@@ -19,36 +19,33 @@ export default Ember.Component.extend({
   btnIcon: Ember.computed.notEmpty("icon"),
 
   @computed("icon", "translatedLabel")
-  btnType(icon, translatedLabel) {
```

Comment:
I dislike this particular example cause you are arguably adding code here with all the `this.` and its slightly less clear.